docs/kilt-baseline-replication.md

```diff
@@ -0,0 +1,136 @@
+# Replication guide KILT
+
+## Setup env
+
+```bash
+# Create a virtual env
+conda create -n kilt37 -y python=3.7 && conda activate kilt37
+
+# Get pyserini scripts and install package
```

Comment:
Can you just say, we need the development installation, see instructions here:
https://github.com/castorini/pyserini/#development-installation